### PR TITLE
Modify short and flags for docker inspect

### PR DESCRIPTION
**Pull request by yuexiao-wang**
Modify short and flags for docker inspect which is inconsistent with inspect.md

Signed-off-by: yuexiao-wang wang.yuexiao@zte.com.cn

### CODE REVIEW

File: cli/command/system/inspect.go [hunk 1]
```diff
@@ -37,7 +37,7 @@ func NewInspectCommand(dockerCli *command.DockerCli) *cobra.Command {
 	flags := cmd.Flags()
 	flags.StringVarP(&opts.format, "format", "f", "", "Format the output using the given go template")
 	flags.StringVar(&opts.inspectType, "type", "", "Return JSON for specified type")
-	flags.BoolVarP(&opts.size, "size", "s", false, "Display total file sizes if the type is container")
+	flags.BoolVarP(&opts.size, "size", "s", false, "Display total file sizes if the type is container values are \"image\" or \"container\" or \"task\"")
```
Comment (thaJeztah) [inline cli/command/system/inspect.go]: Should probably be:

```
Display total file sizes if the type is \"image\", \"container\", or \"task\"
```

**Review by thaJeztah — COMMENTED** [review]
Thanks @yuexiao-wang, left a comment

**Comment by yuexiao-wang** [comment]
@thaJeztah PTAL
I have applied your suggestion to the commit and updated this PR.
Thanks a lot

**Review by vdemeester — APPROVED** [review]
LGTM 🐸

**Comment by yuexiao-wang** [comment]
@thaJeztah PTAL
Thanks!

**Comment by thaJeztah** [comment]
Oh, thanks for the ping; I meant to check this because the length of the output is now really long;

```
Usage:	docker inspect [OPTIONS] NAME|ID [NAME|ID...]

Return low-level information on one or multiple containers, images, volumes, networks, nodes, services, or tasks identified by name or ID.

Options:
  -f, --format string   Format the output using the given go template
      --help            Print usage
  -s, --size            Display total file sizes if the type is "image", "container" or "task"
      --type string     Return JSON for specified type
```

And (afaik) we still strive to get the output to fit within a 80 / 90-chars terminal

I wonder if we should wrap the long lines (and not sure how to deal with the ` --size` flag description in that case

ping @duglin any suggestions here?

Also; did we disable the check for output width?

**Review by mdlinville — APPROVED** [review]
LGTM after @thaJeztah's line wrapping comments are addressed.

**Comment by yuexiao-wang** [comment]
@duglin Could you give me some suggestion?
Thanks!

**Comment by duglin** [comment]
@yuexiao-wang sorry, I missed @thaJeztah's ping.  How about:
```
Display total file size (image,container,task)
```
that fits within 80 columns.

**Comment by yuexiao-wang** [comment]
@duglin Thanks for your review and suggestion!
@thaJeztah's questions  :
1. Output of "Short" for the command docker inspect is so long more than 90-chars
2. not sure how to deal with the --size flag description in that case (Display total file sizes if the type is       "image", "container" or "task")

Therefore I suggest that two solution is as follows:
1.  Output of "Short"  is divided to two lines
2.  Remove "values are "image" or "container" or "task" for the option "-s" . 
     Because only support "-s"  to show total file size  when the type is container in code.
       cc @vdemeester PTAL
   
@thaJeztah  PTAL

**Comment by yuexiao-wang** [comment]
@thaJeztah Could you have any suggestion about the updated PR?
Thanks!

File: docs/reference/commandline/inspect.md [hunk 2]
```diff
@@ -13,14 +13,13 @@ parent = "smn_cli"
 ```markdown
 Usage:  docker inspect [OPTIONS] NAME|ID [NAME|ID...]
 
-Return low-level information on one or multiple containers, images, volumes,
-networks, nodes, services, or tasks identified by name or ID.
+Return low-level information on docker object(s) (e.g. container, image, volume,
+network, node, service, or task) identified by name or ID
```
Comment (thaJeztah) [inline docs/reference/commandline/inspect.md]: Can you update the man page as well? https://github.com/docker/docker/blob/master/man/docker-inspect.1.md

**Review by thaJeztah — CHANGES_REQUESTED** [review]

File: docs/reference/commandline/inspect.md (hunk 2 above)
Comment (yuexiao-wang) [inline docs/reference/commandline/inspect.md]: @thaJeztah PTAL
Have done!

**Review by thaJeztah — APPROVED** [review]
LGTM

**Pull request merged**